pylint_odoo/misc.py

```diff
@@ -112,14 +114,41 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
+        base_path = os.path.dirname(os.path.realpath(__file__))
+        ignores = []
+        ignore_argvs = getopt.getopt([argv for argv in sys.argv[1:]
```

Comment:
Parameters are received with sys.argv but when run with setup.py or tox that variable is empty.
Could you help me with this @moylop260 ?